gnosis/safe-react
# Feature: Use separate rpc endpoint in safe apps

## What it solves
Resolves #2558 
Also, it changes the main's RPC endpoint for polygon to point to infura

## How this PR fixes it
- introduces a new config property `safeAppsRpcServiceUrl`
- adds a new env variable `REACT_APP_SAFE_APPS_RPC_TOKEN` (currently unset) for safe apps RPC token


## How to test it
1. Use any safe app, it should work
2. Configure the interface to use an endpoint of your choice for apps (by editing the config file, all apps request should go through it)
